app/server/src/models/annotation/annotation.model.js

```diff
@@ -1,76 +1,53 @@
 const mongoose = require("mongoose");
-const { Body, createBody } = require("./body.model");
-const { Selector, createTextQuoteSelector, createTextPositionSelector } = require("./selector.model");
-const { Target, createTarget } = require("./target.model");
 
-const annotationSchema = new mongoose.Schema({
-  "@context": {
+const annotationSchema = new mongoose.Schema(
+  {
+    "@context": String,
     type: String,
-    required: true,
-  },
-  type: {
-    type: String,
-    required: true,
-  },
-  body: [
-    {
-      type: mongoose.Schema.Types.ObjectId,
-      ref: "Body",
+    body: [
+      {
+        _id: false,
+        purpose: String,
+        type: String,
```

Comment:
are we sure mongo understands this type as type field or its special mongo type fıeld